doc/stdenv/cross-compilation.chapter.md

```diff
@@ -44,7 +44,14 @@ The exact schema these fields follow is a bit ill-defined due to a long and conv
 
 `config`
 
-: This is a 3- or 4- component shorthand for the platform. Examples of this would be `x86_64-unknown-linux-gnu` and `aarch64-apple-darwin14`. This is a standard format called the "LLVM target triple", as they are pioneered by LLVM. In the 4-part form, this corresponds to `[cpu]-[vendor]-[os]-[abi]`. This format is strictly more informative than the "Nix host double", as the previous format could analogously be termed. This needs a better name than `config`!
+: This is a 3-, 4-, or 5- component shorthand for the platform.  Examples of this would be `x86_64-unknown-linux-gnux32`, `aarch64-apple-darwin14`, and `mips64el-unknown-linux-muslabin32`.  This is a standard format called the "[multiarch tuple](https://wiki.debian.org/Multiarch/Tuples)", as [pioneered by autoconf](https://www.gnu.org/software/autoconf/manual/autoconf-2.65/html_node/System-Type.html#System-Type), [disambiguated to create multiarch](https://wiki.debian.org/Multiarch/Tuples#Used_solution), and adopted by LLVM.  In the 5-part form, this corresponds to `[cpu]-[vendor]-[os]-[libc][abi]`; note that there is no hyphen separating the `[libc]` field from the `[abi]` field. This format is strictly more informative than the "Nix host double", as the previous format could analogously be termed.  This needs a better name than `config`!
+
+"Multiarch tuple" means exactly the same thing as "autoconf tuple" except in [two specific cases](https://wiki.debian.org/Multiarch/Tuples) dealing with 32-bit architectures:
+
+1. Autoconf has multiple `[cpu]` fields for 32-bit x86 systems (`i386-`, `i486-`, `i586-`, and `i686-`).  Multiarch uses `i386-` for all of them.
```

Comment:
This is a good example why it's bad to use multiarch as a reference point. This is incorrect for nixpkgs — nixpkgs uses `i686` for all x86 32 bit CPUs.

Reply:
I have removed that text.  I guess when it comes to x86_32 we aren't following either autoconf or multiarch.